engine/src/tangl/mechanics/sandbox/handlers.py

```
@@ -210,6 +454,43 @@ def _sandbox_inventory(location: SandboxLocation) -> set[str]:
+        raise ValueError("sandbox location has no player asset holder")
+    return AssetTransactionManager().give_asset(player_assets, location, asset_label)
+
+
+def _set_asset_lit(location: SandboxLocation, asset_label: str, lit: bool) -> Token:
+    player_assets = _player_asset_holder(location)
+    if player_assets is None:
+        raise ValueError("sandbox location has no player asset holder")
+    asset = player_assets.get_asset(asset_label)
+    if asset is None:
+        raise KeyError(asset_label)
+    setattr(asset, "lit", lit)
```

Ruff (0.15.12)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)